designs/administrator/4.6/tekton-triggers/index.md

```diff
@@ -0,0 +1,31 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# Tekton Triggers
+
+## Adding Pipelines, Tasks, and Triggers pages under Pipelines
+On the left navigation, the user now has access to Pipelines, Tasks, and Triggers.
+
+Pipelines is now a tabbed page with Pipelines & Pipeline Runs. This allows the user to get to a list of pipeline definitions and the execution of them.
+
+### Pipelines page with Pipelines and Pipeline Runs tabs
+![pipelines-page](img/pipelines-page.jpg)
+- When the user clicks into Tasks, they now have access to Tasks via the Tasks and Task Runs tabs
+- Under the Pipeline Runs tab, we should have 3 columns in the table: Name, Namespace, and Created. The button under Pipeline Runs is “Create Pipeline Runs”
+- The Dev Preview badge should be included on the top right of the page
+
+### Tasks page with Tasks and Task Runs tabs
```

Comment:
As discussed on slack some time ago, Cluster Tasks needs to be here as well. It however might cause some problems with the design. On Cluster-based resources we do not show the Secondary Masthead (project selector) because it doesn't make sense (they are not limited to a cluster).

Perhaps we disable it? Hide it? Leave it alone and the page is just unaffected by it? I'm not sure if the solution is straight forward.